apps/website/screens/components/card/overview/CardOverviewPage.tsx

```diff
@@ -0,0 +1,195 @@
+            can be placed either on the left or right side of the card depending on layout needs.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <strong>Interactive elements</strong> <em>(Optional)</em>: components such as buttons or icons that allow
+            users to take actions related to the card's content.
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+        <DxcParagraph>
+          <em>
+            ** Please note that while these elements are not included by default in the component's configuration, they
+            are the common components in a card. While cards support optional elements like images and actions, it's
+            essential that the overall composition remains visually balanced and aligned with the design system's
+            structure.
+          </em>
+        </DxcParagraph>
+      </>
+    ),
+  },
+  {
+    title: "Placing content in a Card",
+    content: (
+      <>
+        <DxcParagraph>
+          Cards are designed to accommodate flexible layouts and diverse content types. You can include any combination
+          of media and interactive elements, but it's important to follow consistent design patterns for usability and
+          clarity.
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            <strong>Image placement:</strong> Cards support placing the image either on the left or the right side of
+            the layout. The image should maintain a fixed ratio and size for visual harmony. By default, the component
+            provides layout options where the image can appear on the left or right side of the content. However,
+            alternative layouts —such as vertical image placements— can be achieved by placing the image directly within
+            the custom content area. This allows for more flexibility while still adhering to spacing and alignment
+            guidelines.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <strong>Text content:</strong> Titles, descriptions, metadata, or status labels are typically placed in the
+            content area with a clear hierarchy—starting with a bold title, followed by supporting text.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <strong>Interactive elements:</strong> Place buttons, links, or icons in the lower section of the content
+            area or aligned to the end of the card to support related user actions.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <strong>Avoid overloading:</strong> Keep the content concise to prevent visual clutter. A well-structured
+            card should be easy to scan and act upon.
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+        <Image src={example} alt="Card's example" />
+      </>
+    ),
+  },
+  {
+    title: "Best practices",
```

Comment:
Use the best practices format from other components!

![image](https://github.com/user-attachments/assets/955c29cb-0ec4-4a52-af40-5d571ce09bf8)

Remember to use lower case 😃